### PR TITLE
[terra-popup] Fixed accessibility issues with keyboard navigation on up and down arrow key

### PR DESCRIPTION
**Pull request by SG067724**
### Summary

**What was changed:**
Fixed accessibility issues with keyboard navigation on up and down arrow key

**Why it was changed:**
Jaws announced extra text on pressing up and down arrow key when the popup is open.

### Testing

This change was tested using:

- [ ] WDIO
- [x] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9335 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-application-navigation/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Changed
+  Updated jest test case
+
```
Comment (sdadn) [inline packages/terra-application-navigation/CHANGELOG.md:5]: Test changes should not be added to the CHANGELOG.
 
```suggestion
```

File: packages/terra-popup/src/Popup.jsx [hunk 2]
```diff
@@ -276,8 +276,10 @@ class Popup extends React.Component {
     /* eslint-enable no-unused-vars */
 
     if (!isOpen) {
+      document.querySelector('#root').removeAttribute('inert');
       return null;
     }
+    document.querySelector('#root').setAttribute('inert', 'true');
```
Comment (sdadn) [inline packages/terra-popup/src/Popup.jsx:282]: How was this change tested?

File: packages/terra-application-navigation/tests/jest/ApplicationNavigation.test.jsx [hunk 3]
```diff
@@ -5,6 +5,10 @@ import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
 import { mountWithIntl } from 'terra-enzyme-intl';
 import ApplicationNavigation from '../../src/ApplicationNavigation';
 
+const mockRootElement = document.createElement('div');
+mockRootElement.setAttribute('id', 'root');
+document.body.appendChild(mockRootElement);
+
```
Comment (sdadn) [inline packages/terra-application-navigation/tests/jest/ApplicationNavigation.test.jsx:11]: What is the purpose of this mock?

File: packages/terra-popup/src/Popup.jsx [hunk 4]
```diff
@@ -196,6 +211,14 @@ class Popup extends React.Component {
     this.arrowNode = node;
   }
 
+  addInertAttribute = () => {
+    document.querySelector('#root').setAttribute('inert', 'true');
+  };
+
+  removeInertAttribute = () => {
+    document.querySelector('#root').removeAttribute('inert');
+  };
```
Comment (cm9361) [inline packages/terra-popup/src/Popup.jsx:220]: Why are we adding the inert attribute?  This is not supported on all required browsers such as IE 11.  In addition, it does not seem correct to add to the root level.

**Pull request closed without merging**